test/unittests/test_excel_to_json_lists.py

```diff
@@ -47,21 +47,14 @@ def test_expand_lists_from_excel(self) -> None:
         lists_without_excel_reference_output = e2l.expand_lists_from_excel(lists_without_excel_reference)
         self.assertListEqual(lists_without_excel_reference, lists_without_excel_reference_output)
 
-    def test_make_json_lists_from_excel(self) -> None:
-        paths = [f"testdata/excel2json/lists-multilingual/{lang}.xlsx" for lang in ["de", "en", "fr"]]
-        lists_multilingual_output = e2l._make_json_lists_from_excel(paths)  # pylint: disable=protected-access
-        with open("testdata/excel2json/lists-multilingual-output-expected.json", encoding="utf-8") as f:
-            lists_multilingual_output_expected = json.load(f)
-        self.assertListEqual(lists_multilingual_output, lists_multilingual_output_expected)
-
     def test_validate_lists_section_with_schema(self) -> None:
         with open("testdata/excel2json/lists-multilingual-output-expected.json", encoding="utf-8") as f:
             lists_section_valid = json.load(f)
 
-        # validate the valid "lists" section in a correct way
+        # validate the valid "lists" section: should not raise an error
```

Comment:
From here on, I just improve the readability of this file, without substantial changes